demos/k8s-demo/README.md

```diff
@@ -1,303 +1,142 @@
-
-### Steps for the admin-level user
-
-The following steps would be taken by an admin-level user, who has the ability to create and configure a database and to add secret values to a secret store.
-
-These steps make use of the **admin_config.sh** file, which stores the database connection info for the PostgreSQL backend.
-
-#### 1. Provision database
-
-+ Provision protected resources
-
-  **[Option 1] PostgreSQL inside k8s**
-
-  Run the following script to deploy a PostgreSQL instance  using a **StatefulSet**
-  in the **quick-start-backend-ns** namespace:
+## Steps for Security Admin
 
-  ```bash
-  ./01_create_db.sh
-  ```
-    ```
-    >>--- Clean up quick-start-backend-ns namespace
-    Error from server (NotFound): namespaces "quick-start-backend-ns" not found
-    namespace "quick-start-backend-ns" created
-    Ready!
-    secret "quick-start-backend-certs" created
-    >>--- Create database
-    statefulset "pg" created
-    service "quick-start-backend" created
-    Waiting for quick-start-backend to be ready
-    ...
-    Ready!
-    CREATE DATABASE
-    ```
-    Note that the PostgreSQL instance is deployed configured for SSL. We provide
-    simple test certificates in this demo that we upload to the PostgreSQL container
-    using Kubernetes Secrets. In practice, you will have your own certificates; for
-    guidelines on creating the certificates for your PostgreSQL instance, check out
-    the [PostgreSQL documentation](https://www.postgresql.org/docs/9.6/ssl-tcp.html).
+As security admin, you will:
 
-  **[Option 2] Remote PostgreSQL server**
+1. Create a PostgreSQL database
+1. Create a DB user for the application
+1. Add that user's credentials to Kubernetes Secrets
+1. Configure Secretless to connect to PostgreSQL using those credentials
 
-  + Ensure your Kubernetes cluster is able to access your remote DB.
-  + Ensure your remote instance supports SSL
-  + Ensure the remote instance has a database called **quick_start_db**
-  + Update the `DB_` env vars in **./admin_config.sh**. For example (with Amazon RDS):
+To perform all these steps in one go, run:
 
-    ```bash
-    DB_URL=quick-start-backend-example.xyzjshd3bdk3.us-east-1.rds.amazonaws.com:5432/quick_start_db
-    DB_ADMIN_USER=quick_start_db
-    DB_ADMIN_PASSWORD=quick_start_db
-    DB_USER=quick_start_user
-    DB_INITIAL_PASSWORD=quick_start_user
-    ```
-
-#### 2. Configure database and add credentials to secret store
-
-In this step, we will:
-
-+ Configure the protected resources for usage by application (i.e. create DB user, add tables, etc.)
-+ Add the application's access credentials for the database to a secret store
-
-Run:
 ```bash
-./02_configure_db.sh
-```
-
-```
->>--- Set up database
+./01_security_admin_steps
+```
+
+<p></p>
+<details>
+  <summary>View expected output</summary>
+  <pre>
+>>--- Create a new namespace
```

Comment:
I like this better with the spacing you added - when I run it, it looks like:
```
>>--- Create a new namespace

namespace "quick-start-backend-ns" created


>>--- Add certificates to Kubernetes Secrets

secret "quick-start-backend-certs" created


>>--- Create StatefulSet for Database

statefulset "pg" created
service "quick-start-backend" created
Waiting for quick-start-backend to be ready........Done

>>--- Create Application Database

CREATE DATABASE


>>--- Create Database Table and Permissions

CREATE ROLE
CREATE TABLE
GRANT
GRANT


>>--- Store DB credentials in Kubernetes Secrets

namespace "quick-start-application-ns" created
secret "quick-start-backend-credentials" created


>>--- Create Application Service Account

serviceaccount "quick-start-application" created
role "quick-start-backend-credentials-reader" created
rolebinding "read-quick-start-backend-credentials" created


>>--- Create and Store Secretless Configuration

configmap "quick-start-application-secretless-config" created
```

Reply:
yes.  the problem is that gh ignores the pre tags and adds `<p>` around each section, screwing up the formatting.  i was unable to figure out a way around this, despite fiddling with it for a while.  i could use screen shots of the terminal?  wdyt?